services/gen_job/results/output.md

```diff
@@ -2,37 +2,48 @@ Input 1:
 Generated Job Expression:
 ```javascript
 fn(state => {
-  const { data } = state;
-  const visits = data.visits || [];
-  
-  const visitsWithIHS = visits.filter(visit => visit.ihs_number);
-  const visitsWithoutIHS = visits.filter(visit => !visit.ihs_number);
-  
+  const visits = state.data; // Assuming state.data is an array of visit events
+
+  const visitsWithIHS = visits.filter(visit => visit.ihs_number); // Filter visits with IHS number
+  const visitsWithoutIHS = visits.filter(visit => !visit.ihs_number); // Filter visits without IHS number
+
   state.data = {
     visitsWithIHS,
     visitsWithoutIHS
   };
-  
+
   return state;
 });
 ```
+
 ==================================================
 
 Input 2:
```

Comment:
Is it possible to inject the input instruction in here, to make reviewing easier?